src/components/Match/renderMatch.js

```diff
@@ -250,28 +247,33 @@ function renderMatch(m) {
  * Generates data for c3 charts in a match
  **/
 function generateGraphData(match) {
-  // compute graphs
-  let goldDifference = ['Gold'];
-  let xpDifference = ['XP'];
-  goldDifference = goldDifference.concat(match.radiant_gold_adv);
-  xpDifference = xpDifference.concat(match.radiant_xp_adv);
-  const time = ['time'].concat(match.players[0].times);
-  const data = {
-    difference: [time, xpDifference, goldDifference],
-    gold: [time],
-    xp: [time],
-    lh: [time],
-  };
-  match.players.forEach((p, i) => {
-    let hero = heroes[p.hero_id] || {};
-    hero = hero.localized_name;
-    data.gold.push([hero].concat(p.gold_t));
-    data.xp.push([hero].concat(p.xp_t));
-    data.lh.push([hero].concat(p.lh_t));
-  });
-  return data;
+  if (match.players) {
+    // compute graphs
+    let goldDifference = ['Gold'];
+    let xpDifference = ['XP'];
+    goldDifference = goldDifference.concat(match.radiant_gold_adv);
+    xpDifference = xpDifference.concat(match.radiant_xp_adv);
+    const time = ['time'].concat(match.players[0].times);
```

Comment:
instead of doing all these concats, you could do 
```js
const goldDifference = ['Gold', ...match.radiant_gold_adv];
```
this isn't necessary but is a little more concise and easier to understand.